_hackbars/Mannan_Bansal.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: hackbar
+title: STOP DEFORESTATION
+author: Mannan Bansal
+---
+
+OpenSource is full of new innovations and learning opportunities. What are you waiting for, start contributing.
+
+---
+
+## Humans are the only animal who will cut tree, make paper and write save trees on them. #Hacktoberfest #SaveEarth #StopDeforestation
+
+
+![image_caption](../assets/images/Mannan_Bansal.jpg)
```

Comment:
The path is relative to the website not only to the directory. Also, don't forget to put a caption.

```suggestion
![image_caption]({{site.baseurl}}/assets/images/Mannan_Bansal.jpg)
```

Reply:
Ok I will update it.

Reply:
Hey Buddy I have query when I put the path that way it just does not render when we preview the md file. Is it Normal or am I doing something wrong?

Reply:
The `.md` file doesn't render itself. The website uses Jekyll to generate the static page. And the paths are relative to the root website.